packages/core/src/file-context.ts

```diff
@@ -52,6 +53,31 @@ export class FileContext {
     return this._ignoredChecks;
   }
 
+  get ignoredChecksByLine(): Map<number, string[]> {
+    const sourceFile = ts.createSourceFile(this.filename, this.content, ts.ScriptTarget.ES5, true);
+
+    if (!sourceFile) return this._ignoredChecksByLine;
```

Comment:
nit: If we've already processed this file and build up `_ignoredChecksByLine`, we should return it here instead of iterating over sourcefile again.

Reply:
Fixed it in the next iteration.